requirements.txt

```diff
@@ -1,10 +1,10 @@
 wheel>=0.23.0
 numpy>=1.9.2
 scipy>=0.18.0
-pandas>=0.17.0,<0.24.0a
+pandas>=0.17.0,<0.24.0a; python_version <= "3.5"
+pandas>=0.17.0; python_version > "3.5"
```

Comment:
Think you might have an accidental duplicate here - don't think the <0.24 restriction is needed anymore?

Reply:
Thanks for the review!  It's been a while, but I think I had it that way because 0.24 was the last version of pandas to support Python 2, iirc; I didn't do much testing in a Python 2 environment, but I thought it best to keep that version pinned for best compatibility in case people were still using it.